ios/NativeSigner/EthkeyBridge.swift

```diff
@@ -65,6 +65,17 @@ class EthkeyBridge: NSObject {
     resolve(hash)
   }
 
+  @objc func blake2s(_ data: String, resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void {
+    var error: UInt32 = 0
+    var data_ptr = data.asPtr()
+    let hash_rust_str = blake2s(&error, &data_ptr)
```

Comment:
@maciejhirsz maybe a dumb question, not sure if this is a bug on my end but when trying to compile this atm I'm getting an error: `Use of 'blake2s' nearly matches global function 'blake2s' in module '__ObjC' rather than instance method 'blake2s(_:resolve:reject:)'`?

Reply:
Most likely my end since I haven't tried to compile it, I'll rename stuff.